src/Makefile

```diff
@@ -63,7 +63,7 @@ $(CONWAY_TEX_FILES): $(CONWAY_TEX_DIR)/%.tex: %.lagda
 		$(PYTHON) $(HL_SCRIPT) $@ $@ $(HL_LIST); \
 	fi
 
-define latexToPdf =
+define latexToPdf
```

Comment:
I'm surprised that this ever worked! Did this cause a warning or something?

Reply:
For me the make target just silently did nothing. It took some experimenting to figure out that this was the cause.

Reply:
Oh, are you on a mac? I remember having this issue, and my thought was that `define` is probably just not supported there.

Reply:
Yes, on mac, and also not running in nix, so it might be that my make is old? The docs are conflicted:

This one has the `=` sign:

https://www.gnu.org/software/make/manual/make.html#Canned-Recipes

and this one doesn't:
https://ftp.gnu.org/old-gnu/Manuals/make/html_node/make_71.html

Hopefully removing the `=` sign doesn't break it for other people!

Reply:
Yeah, that was the context in which I didn't work for me as well. My thought was that macos might come with some non-GNU `make` that just doesn't support a non-standard feature. I've definitely had that with other standard tools before. But my `make --help` says `Report bugs to <bug-make@gnu.org>` so it may indeed be a version difference. It always worked with `nix` because that just gets all its tools from `nix` anyway.

Anyway, it's nice that it now also works outside of `nix`. And since it didn't break CI it probably doesn't break anything in `nix` elsewhere.